_transactions.md

```diff
@@ -114,7 +114,7 @@ In step one, one prepares the transaction by specifying:
 - The _currency_ to denominate the transaction by.
 - The _amount_ of value to send in the denominated currency.
 - The _origin_ of the transaction can be an account id in the case of a _deposit_.
-- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id, an application id or an Uphold user email.
+- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id, or an application id.
```

Comment:
```suggestion
- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id or an application id.
```

Reply:
You're note a fan of the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma), I see. 😄 No problem, I'll change.

Reply:
That too 😄. But it was mostly about copy consistency.